### PR TITLE
Deletes /ovm and moves its contents into /rollup-contracts

### PR DESCRIPTION
**Pull request by kfichter**
## Description
We have various files (mostly tests) sitting in `/ovm`. This PR moves these files into `/rollup-contracts` and cleans up `/rollup-contracts` in general. I've kept some of the older files from `/ovm/src` for the sake of simplicity, but it's likely they'll need to either be deleted or moved into a different package. I figured we could handle that as part of a separate ticket in the future in order to keep this PR as tightly scoped as possible.

Tests are passing, though we have some linting errors that I need to fix.

### Fixes
- Fixes [YAS-443](https://optimists.atlassian.net/browse/YAS-443)

## Contributing Agreement

- [x] I have read and understood the [Optimism Contributing Guide and Code of Conduct](https://github.com/ethereum-optimism/optimism-monorepo/blob/master/.github/CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

File: packages/ovm/test/bridge/L1ToL2TransactionPasser.spec.ts [hunk 1]
```diff
@@ -1,29 +0,0 @@
...
-    wallet = getWallets(provider)[0]
-    l1ToL2TransactionPasser = await deployContract(
-      wallet,
-      L1ToL2TransactionPasserContractDefinition
-    )
-  })
-  const entryPoint = '0x0000000000000000000000000000000000001234'
-  const callData = '0xdeadbeefee5555'
-  it('should emit the correct L1ToL2Transaction L1 event when an L1->L2 tx is sent', async () => {
-    await l1ToL2TransactionPasser
```
Comment (karlfloersch) [inline packages/ovm/test/bridge/L1ToL2TransactionPasser.spec.ts:24]: Are these tests reflected anywhere in these changes or just generally deleted? Do these contracts impose a circular dependencies? If they do maybe putting stuff in `rollup-full-node` makes sense? Although ideally these tests are just in the `rollup-contracts` package

**Comment by kfichter** [comment]
I'm going to go ahead and close this PR since it's mostly covered in #166

**Comment by kfichter** [comment]
@karlfloersch I'll address your comments in the other PR.

File: packages/ovm/.soliumrc.json [hunk 2]
```diff
@@ -1,21 +0,0 @@
...
-    "indentation": [
-      "error",
-      4
-    ],
-    "linebreak-style": [
-      "error",
-      "unix"
-    ]
-  }
-}
```
[the remark this answers is not in the document]
Comment (kfichter) [reply packages/ovm/.soliumrc.json:21]: Ohhh yes we should. Good catch.

File: packages/ovm/test/bridge/L1ToL2TransactionPasser.spec.ts (hunk 1 above)
Comment (kfichter) [reply packages/ovm/test/bridge/L1ToL2TransactionPasser.spec.ts:24]: These tests have all been transferred over to the `rollup-contracts` pacakge.

**Comment by kfichter** [comment]
>   * TLDR, rename package to contracts? rename `rollup` directory to `optimistic-ethereum`?

Yeah, I think renaming this package is a good idea. I'll go ahead and make that change.

> * instead of `execution` (even though I find it to be a very descriptive name) we could consider calling that directory `ovm` to help people find the contracts that they're looking for if they come having heard about this `ovm` thing that we've built.
> * moving `SafetyChecker.sol` into the `ovm` package (currently called `execution`) could be good because it feels the `ExecutionManager` is sooo closely tied to the `SafetyChecker`, and I can't really see anywhere other in our crazy use-case where anyone would be trying to use our safety checker. That said, I guess _technically_ it could be generally useful as we did steal the idea of safety checking from the original 'purity checker' in Casper FFG. Ok so I don't actually have a strong opinion here.
> * `precompiles` could actually be a directory under `ovm` (`ovm/precompiles`) because they are really only used in the context of the ExecutionManager.

I like all of these!
